frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -63,6 +64,8 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
   serial,
   rdp,
   LoadingComponent,
+  type,
+  showActions = true,
```

Comment:
this name is a bit misleading, since we have multiple "actions" on this page, but it hides only the `Open In New Window` action